app/services/billable_metrics/prorated_aggregations/unique_count_service.rb

```diff
@@ -196,6 +196,15 @@ def added_and_removed_list
           )
         end
       end
+
+      def recurring_value
+        previous_charge_fee_units = previous_charge_fee&.units
+
+        return previous_charge_fee_units if previous_charge_fee_units
+
+        recurring_value_before_first_fee = prorated_persisted_query.sum("(#{persisted_pro_rata})::numeric")
+        recurring_value_before_first_fee <= 0 ? nil : recurring_value_before_first_fee
```

Comment:
Style/TernaryParentheses: Use parentheses for ternary expressions with complex conditions.